igvm/hypervisor.py

```diff
@@ -730,8 +731,11 @@ def umount_temp(self, device_or_path):
     def remove_temp(self, mount_path):
         self.run('rmdir {0}'.format(mount_path))
 
-    def format_storage(self, device):
-        self.run('mkfs.xfs -f {}'.format(device))
+    def format_storage(self, device, os):
```

Comment:
Wouldn't it be cleaner to give only True/False here for ftype parameter and determine it above?